pkg/apis/aerogear/v1alpha1/types.go

```diff
@@ -246,8 +246,9 @@ type GenericStatus struct {
 	Message  string      `json:"message"`
 	Attempts int         `json:"attempts"`
 	// marked as true when all work is done on it
-	Ready   bool   `json:"ready"`
-	Version string `json:"version"`
+	Ready           bool   `json:"ready"`
+	Version         string `json:"version"`
+	OperatorVersion string `json:"operator"`
```

Comment:
shouldn't the `OperatorVersion` be added to `installation_type.go`?

Reply:
Yes - going to review this in a follow-on PR to see if necessary, and will update there if needed